doc/dev/api/server/Ppx_syntax.wiki

```diff
@@ -0,0 +1,67 @@
+=RPC ppx syntax extension
+
+The ocsigen-start syntax extension allows to write, in one function, both the
+client and server part of your RPC.
+
+
+==Sections
+
+The location of code execution is specified by section annotations. 3 types
```

Comment:
"The location of code execution is specified by section annotations."

This is not the case for rpcs. You cannot choose the location of your code.

Reply:
Also, 
3 => Three